### DISCUSSION TITLE
Vehicle  destroyed

### DISCUSSION THREAD

**Discussion by jus61** [Vdauphin/HeartsAndMinds#1510] [Question and answer] [answered]
I'm not sure if it's a mistake on my part so I'll just ask!

If an own vehicle is destroyed by a mine, it is automatically deleted after burning out. The map marker remains on the map.

All vehicles are in the group btc_vehicles. [1.23.3]

**Comment by Vdauphin** [comment]
Hello,

> it is automatically deleted after burning out. 

Yes this is wrong, something is deleting your vehicles

Cheers

**Comment by jus61** [comment]
My English is not the best ;) do you have any idea how to solve the problem ?

**Reply by Vdauphin** [reply]
Just download the mission here: https://github.com/Vdauphin/HeartsAndMinds/releases and don't change anything
And load CBA and ACE only

You can join the official A3 Discord for more help

**Comment by jus61** [comment]
I have not changed anything on the mission

**Reply by Vdauphin** [reply, marked as answer]
Then it is may be mods you loaded or your Zeus who deleted the vehicle

**Comment by jus61** [comment]
The problem was caused by the LOGIC FSM mod !
